woocommerce/woocommerce-ios
# [Unified Order Editing] Refactor Order Creation form

Closes: https://github.com/woocommerce/woocommerce-ios/issues/6961

## Description

This PR refactors `NewOrder` and `NewOrderViewModel` to have universal naming for Create and Edit flows.
In addition, it presents different sync error notices.

## Testing

1. Go to the Order tab, open any existing order.
2. Wait until order fully loads. Tap "•••" in navbar. Select "Edit".
3. Disable network.
4. Try updating product list or customer data.
5. Confirm correct notice is displayed.
6. Enable network.
7. Hit "Retry" and confirm sync succeeds.

## Screenshots

<img width=350 src=https://user-images.githubusercontent.com/3132438/175298582-2c489d38-301c-48ea-a8d4-a3bf40c1760e.png>

---
